pkgs/by-name/ld/ld64/patches/0020-Use-STL-containers-instead-of-LLVM-containers.patch

```diff
@@ -0,0 +1,2813 @@
+From 0b4c0b560c83884ec893a63395bc37fde9e4e1da Mon Sep 17 00:00:00 2001
+From: Randy Eckenrode <randy@largeandhighquality.com>
+Date: Wed, 13 Nov 2024 13:53:14 -0500
+Subject: [PATCH 20/20] Use STL containers instead of LLVM containers
```

Comment:
Can we split out (or just drop) the file deletions here, since they shouldn’t be necessary to have in Nixpkgs and the huge patch adds to tarball bloat and brings Safari to its knees? :)

Reply:
My fork deletes these files because it’s functionally its own thing, so I don’t want to leave the files there (because they can’t be mixed with LLVM headers). I could delete that section of the patch, but the more manipulations the patch set requires after being generated by `git format-patch`, the more of a maintenance burden it is.

Reply:
Yeah, deleting the files on the branch is fine. But if you just do it in a follow‐up commit, then we can omit it from Nixpkgs without any specific patch munging, since it provides no value to us. (If you’re worried about an increasing number of commits to omit, then it should be simple to, say, tag the relevant commits and filter out those patches (if in the commit subject, this would be a simple filename‐based operation).)

(I’d also be fine just, like, pointing the derivation at your repository, which would obviate all these patch size concerns. But if we’re going to vendor this in Nixpkgs I think it’s reasonable to prioritize not having the patches be unnecessarily huge.)

Reply:
The problem with pointing it at my repository is I don’t tag releases. This is coming from a branch I rebase as I make changes. It’s unstable, and pinning a commit could get GC’d by GitHub. I can split out the deletions into a separate commit and repush.